app/views/components/docs/publisher-metadata.yml

```diff
@@ -89,3 +89,17 @@ examples:
             - <a href='/government/organisations/ministry-of-defence'>Ministry of Defence</a>
     context:
       right_to_left: true
+  with_single_page_notification_button:
+    description: |
+      The component can render with a single page email notification button which allows the user to subscribe to email notifications about the current page. The button should only appear on pages with an update history (therefore it has been set to appear only if `link_to_history` is true).
```

Comment:
Can this be reworded to clarify exactly what difference this makes? If my understanding is correct, it adjusts the bottom margin?